src/config.ts

```diff
@@ -296,43 +296,44 @@ export class MirrordConfigManager {
     if (this.active) {
       // User has selected a config (via active config button).
       new NotificationBuilder()
-        .withMessage("using active mirrord configuration")
+        .withMessage("Using active mirrord configuration.")
         .withOpenFileAction(this.active)
         .withDisableAction("promptUsingActiveConfig")
         .info();
+
       return this.active;
     } else if (config.env?.["MIRRORD_CONFIG_FILE"]) {
       // Get the config path from the env var.
-      return vscode.Uri.parse(`file://${config.env?.["MIRRORD_CONFIG_FILE"]}`, true);
+      const configFromEnv = vscode.Uri.parse(`file://${config.env?.["MIRRORD_CONFIG_FILE"]}`, true);
+
+      new NotificationBuilder()
+        .withMessage(`Using mirrord configuration from env var "MIRRORD_CONFIG_FILE".`)
+        .withOpenFileAction(configFromEnv)
+        .withDisableAction("promptUsingActiveConfig")
```

Comment:
How about giving this case a separate disable action?

Reply:
Added it.